### DISCUSSION TITLE
Best way to suggest how to contact us

### DISCUSSION THREAD

**Discussion by nhsavage** [MetOffice/PyPRECIS#152] [Ideas]
At the moment, the contributing guidelines state  "If in doubt, please contact the PRECIS team ([precis@metoffice.gov.uk](mailto:precis@metoffice.gov.uk)) if you have questions."

Is this now the most appropriate solution ? Would we better to offer the following options:
 1. issue
 2. Pull request
 3. Discussion (here)

Perhaps we could steal some of the wording from the Iris developers guide? 

https://scitools-iris.readthedocs.io/en/stable/developers_guide/contributing_getting_involved.html

**Comment by hsteptoe** [comment]
Personally my vote is to remove reference to the precis mailbox. There has been a lack of clarity about how we use and who checks it for years now (side note: we should consider updating the Met Office website to reflect whatever we decide here, and maybe point to this discussion forum).

I think we should add some clarity about when and why you might use a issue vs pull request vs discussion post into the CONTRIBUTING.md guide
